test/integration/sequelize.test.js

```diff
@@ -1018,6 +1018,27 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
         .to.be.rejectedWith('Database "cyber_bird" does not match sync match parameter "/$phoenix/"');
     });
 
+    it('won\'t create duplicated unique indexes', function() {
```

Comment:
Move this test [here](https://github.com/sequelize/sequelize/blob/83dfc5ae9cf671d448690bc3c8a6d0f5ef45a681/test/integration/model/sync.test.js#L133)

Change title `should not create duplicate unique indexes with unique:true`